index.html

```diff
@@ -230,11 +464,74 @@ <h3>CSS support</h3>
 				</div>
 			</section>
 		</section>
-		<section id="packaging">
+		<section id="ebrl-nav">
+			<h2>Primary entry page</h2>
+
+			<section id="navdoc-intro" class="informative">
+				<h3>Introduction</h3>
+
+				<p>To make it easier to deploy [=eBraille publications=] on the web, the [=EPUB navigation document=] is
+					used as the primary entry point for publications. The required file naming (<code>index.html</code>)
+					and location of this file mean it will be the first document loaded when users browse to a folder
+					containing an eBraille publication.</p>
+
+				<p>By requiring the [=primary entry page=] be located in the [=publication root=], it is also easier for
+					users to unzip an eBraille publication and read it locally, if they so choose. If all the other
+					resources in the publication are placed in a subfolder, for example, the primary entry page will be
+					the first HTML document users encounter in the unzipped folder.</p>
+			</section>
+
+			<section id="nav-req">
+				<h3>General requirements</h3>
+
+				<p>The primary entry page MUST be named <code>index.html</code>. This file has to be located in the
+					[=publication root=], as defined in <a href="#fileset-structure"></a>.</p>
+
+				<p>The primary entry page MUST be a conforming <a data-cite="epub-33#sec-nav">EPUB navigation
+						document</a> [[epub-33]].</p>
+
+				<p>As the primary entry page is meant to help browser users navigate the publication, it SHOULD NOT be
+					included in the <a href="#spine">spine</a>. If the publication needs a table of contents in the
+					spine, it is better to create a separate document with publication-specific formatting.</p>
+			</section>
+
+			<section id="navigation">
+				<h3>Navigation</h3>
+
+				<section id="nav-toc">
+					<h4>Table of contents</h4>
+
+					<p>For compatibility with web-based rendering, the table of contents MUST also be identified by the
+						[^/role^] attribute [[html]] value <a data-cite="dpub-aria#doc-toc"><code>doc-toc</code></a>
+						[[dpub-aria]]</p>
+
+					<aside class="example" title="Table of contents identified by the epub:type and role attributes">
+						<pre><code>&lt;nav epub:type="toc" role="doc-toc">
```

Comment:
Do we really need epub:type="toc" since it doesn't really do anything I would prefer to leave it out unless it is required by legacy reading systems but even then...

Reply:
Yes, if the idea is to have a valid epub file set. If we drop it, then the navigation document doesn't conform.

Reply:
Oh really, that's interesting.  Wow, too bad we didn't change that to use the role="doc-toc" instead.  oh well, good to know. thanks.

Reply:
This has been one of the roadblocks to adopting html in epub. We can't just move to aria because then all reading systems that aren't up to date would fail to load the table of contents. Same applies to moving off epub:type for the navigation document.